utility/src/main/java/org/oppia/android/util/parser/ImageLoader.kt

```diff
@@ -10,12 +11,20 @@ interface ImageLoader {
    * asynchronous operation, and may take a while if the image needs to be downloaded from the
    * internet.
    */
-  fun loadBitmap(imageUrl: String, target: ImageTarget<Bitmap>)
+  fun loadBitmap(
+    imageUrl: String,
+    target: ImageTarget<Bitmap>,
+    vararg transformations: Transformation<Bitmap>
```

Comment:
It would be preferable to help the loader avoid depending on Glide for its contract (since Glide is really more of its implementation). Could we maybe instead take a list of preset transformations? I'm thinking something like:

```kotlin
fun loadBitmap(
  imageUrl: String,
  target: ImageTarget<Bitmap>,
  transformations: List<ImageTransformation> = listOf()
)

...

enum class ImageTransformation {
  BLUR
}
```

Then, we would move the blurring code into ImageLoader which may also simplify upstream code.

Reply:
Ok, I changed that. There's probably more improvements to be made.